daemon/manager/dump/dump.go

```diff
@@ -0,0 +1,49 @@
+package dump
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"kmesh.net/kmesh/pkg/status"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dump",
+		Short: "Dump xds workloads",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Can you add an example on how to use it?

Reply:
Provides a shortcut for curl localhost:15200/debug/config_dump/workload
![image](https://github.com/kmesh-net/kmesh/assets/58595459/7f959e71-d8c1-43b8-9027-227e22612845)

Reply:
> Can you add an example on how to use it?

Sure